cli/dcoscli/data/help/node.txt

```diff
@@ -11,8 +11,9 @@ Usage:
     dcos node diagnostics delete <bundle>
     dcos node diagnostics download <bundle> [--location=<location>]
     dcos node list-components [--leader --mesos-id=<mesos-id> --json]
-    dcos node log [--follow --lines=N --leader --master --mesos-id=<mesos-id> --slave=<agent-id>]
+    dcos node log [--follow --lines=N --leader --master --mesos-id=<mesos-id> --slave=<slave-id>]
```

Comment:
`agent-id` is correct here. We still have `--slave` for deprecation, which we will remove soon.